src/components/Drawer/Drawer.Overview.stories.mdx

```diff
@@ -0,0 +1,552 @@
+import { useRef, useState } from 'react';
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { Drawer } from './Drawer';
+import { Alert } from '../Alert/Alert';
+import { Box } from '../Box/Box';
+import { Button } from '../Button/Button';
+import { TextInput } from '../TextInput/TextInput';
+import { RadioGroup } from '../RadioGroup/RadioGroup';
+
+<Meta title="Components/Drawer/Overview" component={Drawer} />
+
+# Drawer
+
+A Drawer is a panel that slides in from one edge of the viewport and overlays content on top of the page. It contains information or actions without leaving the context of the original page.
+
+<Canvas withSource="open">
+  <Story
+    name="Basic Drawer"
+    parameters={{
+      chromatic: { disableSnapshot: true },
+    }}
```

Comment:
Tell chromatic to ignore these stories because the user has to click on the button to see the Drawer. Drawer.VisualTests.stories.tsx provides coverage of  Drawer snapshots

Reply:
If you want, you could programmatically click to open the drawer.
https://storybook.js.org/docs/react/writing-stories/play-function

Reply:
I tried this but it doesn't play nicely with the "Docs" tab and the code sample that is generated, so opted to rely on Drawer.VisualTests.stories.tsx to capture chromatic snapshots. Works great for the "Canvas" tab though.

Here's what I mean:
![Screen Shot 2022-04-18 at 10 13 36 AM](https://user-images.githubusercontent.com/1447339/163845856-66eb2cb5-7a52-46f5-9aef-f524fb042951.png)